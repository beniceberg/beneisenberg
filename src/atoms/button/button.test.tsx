import React from "react"
import renderer from "react-test-renderer"
import Button from "./button"

describe("Button", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Button title="Default Starter" onClick={() => console.log('a')} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})