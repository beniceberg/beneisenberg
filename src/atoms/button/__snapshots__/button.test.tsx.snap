// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  className="button black"
  onClick={[Function]}
  tabIndex={0}
>
  Default Starter
</button>
`;
